### ISSUE TITLE
Crash when opening Realm table.

### ISSUE THREAD

**Issue by khoren93** [Flipboard/FLEX#111]
Hi, thanks for Realm browser supporting:) 
But when i trying to open any table, app crashes...

*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[__NSArrayM insertObject:atIndex:]: object cannot be nil'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010ead1e65 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x000000010e54adeb objc_exception_throw + 48
	2   CoreFoundation                      0x000000010e9988c5 -[__NSArrayM insertObject:atIndex:] + 901
	3   FLEX                                0x000000010adf0340 -[FLEXTableContentViewController contentAtRow:] + 464
	4   FLEX                                0x000000010adb3e5d -[FLEXMultiColumnTableView tableView:cellForRowAtIndexPath:] + 365
	5   UIKit                               0x000000010bd86e43 -[UITableView _createPreparedCellForGlobalRow:withIndexPath:willDisplay:] + 766
	6   UIKit                               0x000000010bd86f7b -[UITableView _createPreparedCellForGlobalRow:willDisplay:] + 74
	7   UIKit                               0x000000010bd5ba39 -[UITableView _updateVisibleCellsNow:isRecursive:] + 2996
	8   UIKit                               0x000000010bd9001c -[UITableView _performWithCachedTraitCollection:] + 92
.......................

**Comment by TimOliver** [comment]
Hi @khoren93! I'm the one who filed the PR for Realm support! Glad to hear you've started using it!

Oh dear! That was quick! While I tested the library on both a test Realm file, and a Realm file from one of my own apps, it sounds like it doesn't actually handle nullability properly. Sorry I missed that!

It'll take me a while to set up some test Realm data and test this myself, so if you have access to the FLEX codebase, could you please try this code out for me?

In the file `FLEX/GlobalStateExplorers/DatabaseBrowser/FLEXRealmDatabaseManager.m`, on line 104, can you please replace that with this?

```
id value = [result valueForKey:property.name];
entry[property.name] = (value) ? (value) : [NSNull null];
```

That should check to see if there's actually a value provided by that Realm property, and if there's not, insert an `NSNull` instance in its place to ensure that a `nil` value isn't applied to the `NSArray` there.

Please let me know if that fixes it, and if it does, I'll file another PR. Thanks!

**Comment by khoren93** [comment]
Hi @TimOliver! Thats fixes crash, thanks! :)

**Comment by ryanolsonk** [comment]
Fixed in 552e687b9c07bc9147959f8cb10bd62c29675e2d
